index.html

```diff
@@ -1,6 +1,8 @@
 <!doctype html>
 <html>
-  <head>
+  <head><title>
+  The title</title>
```

Comment:
:+1: although you should move `<title>` down to the next line -- not immediately next to `<head>`